From 819156ad4891f37cfcf93bbbb46a0e38a71b89ed Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Petr=20P=C3=ADsa=C5=99?= <ppisar@redhat.com>
Date: Mon, 28 Feb 2011 14:16:59 +0100
Subject: [PATCH] Do not use real domains in warnquota example

Providing private domain names in default warnquota.conf can tend
some users to reply. Owners of that domain could get mad.
---
 warnquota.conf | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/warnquota.conf b/warnquota.conf
index d8b5d5d..172d635 100644
--- a/warnquota.conf
+++ b/warnquota.conf
@@ -5,15 +5,15 @@
 
 # values can be quoted:
 MAIL_CMD 	= "/usr/sbin/sendmail -t"
-FROM 		= 
+FROM 		=  root@example.com
 # but they don't have to be:
 SUBJECT 	= NOTE: You are exceeding your allocated disk space limits
-CC_TO 		= "root@localhost"
+CC_TO 		= "root@example.com"
 # If you set this variable CC will be used only when user has less than
 # specified grace time left (examples of possible times: 5 seconds, 1 minute,
 # 12 hours, 5 days)
 # CC_BEFORE = 2 days
-SUPPORT 	= "root@myhost.com"
+SUPPORT 	= "root@example.com"
 PHONE 		= "(123) 456-1111 or (222) 333-4444"
 # Text in the beginning of the mail (if not specified, default text is used)
 # This way text can be split to more lines
@@ -25,7 +25,7 @@ MESSAGE         = Your disk usage has exceeded the agreed limits\
  on this server|Please delete any unnecessary files on following filesystems:|
 # Text in the end of the mail (if not specified, default text using SUPPORT and PHONE
 # is created)
-SIGNATURE	= root@localhost
+SIGNATURE	= root@example.com
 # Following text is used for mails about group exceeding quotas
 GROUP_MESSAGE  = Hello,|\
 your group %i is using too much disk space at %h.|\
-- 
1.7.11.7

